scalafix-core/shared/src/main/scala/scalafix/rule/ScalafixRules.scala

```diff
@@ -26,7 +26,8 @@ object ScalafixRules {
     NoAutoTupling(index),
     Disable(index, DisableConfig.default),
     DisableUnless(index, DisableUnlessConfig.default),
-    MissingFinal(index)
+    MissingFinal(index),
+    SingleAbstractMethod(index)
```

Comment:
What about `PreferSAM` or `SAM`? Not really sure if those are any better though. Suggestions for another name are welcome! cc/ @gabro 😄

Reply:
`SingleAbstractMethodConversion` or `SAMConversion`

Reply:
In general, the names are `VerbSomething` like `RemoveUnusedImports`. In scalafix, we use `PreferX` and `AvoidX` for several rewrites like `AvoidInfix`

Reply:
I like `PreferSAM` and I bet @fommil likes it too.

Reply:
This SAM approved by Sam.

Reply:
`ConvertSingleAbstractMethod` ?